ARE/components/actuator.knx/src/main/java/eu/asterics/component/actuator/knx/KnxInstance.java

```diff
@@ -1072,4 +1115,148 @@ private void sendKNX(String groupaddress, String type, String value) {
             AstericsErrorHandling.instance.getLogger().severe("KNX connection not open");
         }
     }
+    
+    /**
+     * Checks if the DPT type needs the data formatted as integer or double id
+     * 
+     * @param DPTid	DPT id as String
+     *           
+     * @return DPT 	true when DPT type should be formatted as int
+     * 				false when DPT type should be formatted as double
+     */
+	private boolean DPTDataTypeIsInt(String DPTid) {

```

Comment:
what does this method do? I don't get it why e.g. "5.010" should be converted to integer and "9.026" to double?! whats with all other values, e.g. "9.031"?

Reply:
ok, if these are KNX specific IDs then please do something like this instead:
```
private List KNX_IDS_INTEGER = Arrays.asList({"5.010", "5.001", "5.004", ...});
private List KNX_IDS_DOUBLE = Arrays.asList({"9.026", "9.027", "9.028", ...});
private boolean DPTDataTypeIsInt(String DPTid) {
    return KNX_IDS_INTEGER.contains(DPTid);
}
```
in this case KNX_IDS_DOUBLE even would not be needed.

Reply:
done
KNX_IDS_DOUBLE not needed but implemented as preparation, because it will be needed, when date and time formats are added...

Reply:
done
KNX_IDS_DOUBLE not needed but implemented as preparation, because it will be needed, when date and time formats are added...